src/components/PrefetchBalancesWrapper/PrefetchBalancesWrapper.tsx

```diff
@@ -31,17 +31,23 @@ const hasUnfetchedBalancesAtom = atom<boolean>(true)
 export default function PrefetchBalancesWrapper({
   children,
   shouldFetchOnAccountUpdate,
+  shouldFetchOnHover = true,
   className,
-}: PropsWithChildren<{ shouldFetchOnAccountUpdate: boolean; className?: string }>) {
+}: PropsWithChildren<{ shouldFetchOnAccountUpdate: boolean; shouldFetchOnHover?: boolean; className?: string }>) {
   const { account } = useWeb3React()
   const [prefetchPortfolioBalances] = usePortfolioBalancesLazyQuery()
 
   // Use an atom to track unfetched state to avoid duplicating fetches if this component appears multiple times on the page.
   const [hasUnfetchedBalances, setHasUnfetchedBalances] = useAtom(hasUnfetchedBalancesAtom)
   const fetchBalances = useCallback(() => {
     if (account) {
-      prefetchPortfolioBalances({ variables: { ownerAddress: account, chains: GQL_MAINNET_CHAINS } })
-      setHasUnfetchedBalances(false)
+      // Backend takes <2sec to get the updated portfolio value after a transaction
+      // This timeout is an interim solution while we're working on a websocket that'll ping the client when connected account gets changes
+      // TODO(WEB-3131): remove this timeout after websocket is implemented
+      setTimeout(() => {
+        prefetchPortfolioBalances({ variables: { ownerAddress: account, chains: GQL_MAINNET_CHAINS } })
+        setHasUnfetchedBalances(false)
+      }, 3500)
```

Comment:
could be a little cleaner to just do  setTimeout(fetchBalances, 3500) on L74. Also we tend to do:
```suggestion
      }, ms("3.5s"))
```
for readability

Reply:
hm I think i will keep timeout inside fetchBalances so we don't forget to timeout every time we want to fetch balance